stats/tick.go

```diff
@@ -0,0 +1,24 @@
+package stats
+
+import "time"
+
+// provides "clean" ticks at precise intervals, and delivers them shortly after
```

Comment:
because it's more accurate. it puts the ticks close to the boundaries between stats.
see also https://blog.raintank.io/25-graphite-grafana-and-statsd-gotchas/#statsd.flush.offset